Packs/OpenAI/Integrations/OpenAI/OpenAI.yml

```diff
@@ -1,64 +1,113 @@
+  display: API Version
+  name: version
+  required: false
+  type: 0
+- additionalinfo: Model to use for the integration Test if different from the default model, "text-davinci-003"
+  display: Model to Use for Test
+  name: test_model
   required: false
+  type: 0
+description: This integration leverages the OpenAI and Azure OpenAI REST APIs to generate human-like responses to text prompts. The APIs provide access to OpenAI's powerful language models including the GPT-3.5 model series. These models can be easily adapted to your specific task including but not limited to content generation, summarization, semantic search, and natural language to code translation.
+display: OpenAI
+name: OpenAI
 script:
-  script: ''
-  type: python
   commands:
-  - name: openai-completions
-    arguments:
-    - name: prompt
+  - arguments:
+    - description: Add your question or text
+      name: prompt
+      required: true
+    - auto: PREDEFINED
+      defaultValue: gpt-3.5-turbo
+      description: Name of model to use. For Azure OpenAI, this should be the name of the DEPLOYMENT, not the generic model name.
+      name: model
+      predefined:
+      - gpt-3.5-turbo
+    description: Send prompt to OpenAI ChatGPT
+    name: openai-chatgpt
+    outputs:
+    - contextPath: OpenAI.ChatGPTResponse.ChatGPT Response
+      description: ChatGPT response
+    - contextPath: OpenAI.ChatGPTResponse.Created Time
+      description: Response created time as Unix timestamp
+    - contextPath: OpenAI.ChatGPTResponse.Model
+      description: Model that generated the response
+    - contextPath: OpenAI.ChatGPTResponse.Number of Completion Tokens
+      description: Number of tokens in the completion (response)
+    - contextPath: OpenAI.ChatGPTResponse.Number of Prompt Tokens
+      description: Number of tokens in the prompt (input)
+    - contextPath: OpenAI.ChatGPTResponse.Number of Total Tokens
+      description: Total number of tokens in the completion and prompt
+    - contextPath: OpenAI.ChatGPTResponse.id
+      description: ChatGPT response ID
+  - arguments:
+    - description: Instruction
+      name: prompt
       required: true
-      description: Instruction
-    - name: model
-      auto: PREDEFINED
+    - auto: PREDEFINED
+      defaultValue: text-davinci-003
+      description: The model which will generate the completion. Some models are suitable for natural language tasks, others specialize in code. For Azure OpenAI, this should be the name of the DEPLOYMENT, not the generic model name.
+      name: model
       predefined:
       - text-davinci-003
       - text-curie-001
       - text-babbage-001
       - text-ada-001
       - code-davinci-002
       - code-cushman-001
-      description: The model which will generate the completion. Some models are suitable for natural language tasks, others specialize in code
-      defaultValue: text-davinci-003
-      type: textArea
-    - name: temperature
+    - defaultValue: "0.2"
```

Comment:
Is there a reason this was changed to 0.2?

Reply:
IMO for most security-related use cases, it is more common to want a deterministic, high-probability result. However, the default value is not a big deal, since it can and should be overridden by the user with the appropriate value for their use case. I am fine with changing the default back to 0.7 for consistency.